src/Hng.Application/Features/HelpCenter/Dtos/HelpCenterResponseDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hng.Application.Features.HelpCenter.Dtos
+{
+    public class HelpCenterResponseDto<T>
+    {
+        public int StatusCode { get; set; }
+        public string Message { get; set; }
+        public T Data { get; set; }
+    }
```

Comment:
Please add JsonPropertyName for snake case

Reply:
okaybossithoughtitdidntneedbeforeiwilldoitasap